mne/viz/epochs.py

```diff
@@ -968,13 +968,7 @@ def plot_epochs(
     boundary_times = np.arange(len(epochs) + 1) * len(epochs.times) / sfreq
 
     # events
-    if events is None:
-        warn(
-            "The current default events=None is deprecated and will change to "
-            "events=True in MNE 1.6. Set events=False to suppress this warning.",
-            category=FutureWarning,
-        )
-        events = False
+    _validate_type(events, (bool, np.ndarray), "events")
     if events is False:
```

Comment:
can we get rid of the antipattern now?  I guess `if not events` still won't work because "the truth value of an array is ambiguous".

Reply:
Right, we can't just check truthiness directly. But we do know it's bool or ndarray with the `_validate_type` above so `if events is False` here and `if events is True` below should both do the right thing